app/controllers/users_controller.rb

```diff
@@ -0,0 +1,59 @@
+class UsersController < ApplicationController
```

Comment:
We will see later if UserController is required or not. After devise gem integrated.